app/assets/v2/js/pages/profile.js

```diff
@@ -39,43 +65,146 @@ $(document).ready(function() {
       fetchInProgress = true;
       loadingImg.className = loadingImg.className.replace('hidden', 'visible');
 
-      fetch(location.href.replace(location.hash, '') + '?p=' + (++page) + '&a=' + activityName).then(
-        function(response) {
+      fetch(
+        location.href.replace(location.hash, '') +
+          '?p=' +
+          ++page +
+          '&a=' +
+          activityName
+      )
+        .then(function(response) {
           if (response.status === 200) {
-            response.text().then(
-              function(html) {
-                const results = document.createElement('div');
-
-                activityContainer.setAttribute('page', page);
-                results.insertAdjacentHTML('afterBegin', html);
+            response.text().then(function(html) {
+              const results = document.createElement('div');
 
-                const childs = results.children;
+              activityContainer.setAttribute('page', page);
```

Comment:
hmm are these just linting changes?

Reply:
Yes, its linting. Wasn't removed.

Reply:
@iamonuwa can you please not make arbitrary style changes to the existing code? It makes it difficult to review the PR because its unclear what has actually changed. The less lines of code changed the better, in general.